clip_benchmark/datasets/builder.py

```diff
@@ -535,27 +536,75 @@ def build_tfds_dataset(name, transform, download=True, split="test", data_dir="r
     return ds
 
 
-def build_wds_dataset(dataset_name, transform, download=True, split="test", data_dir="root"):
-    """Load a dataset in WebDataset format (for now only local files). Must be in a .tar or .tar.gz"""
+def build_wds_dataset(dataset_name, transform, download=False, split="test", data_dir="root"):
+    """Load a dataset in WebDataset format. Only local files for now. Expected file structure is:
```

Comment:
I think only local files is wrong now ?

Reply:
You're right, I forgot to change that before